dts/x86/intel/alder_lake.dtsi

```diff
@@ -35,6 +35,15 @@
 		#interrupt-cells = <3>;
 	};
 
+	intc_loapic: loapic@fee00000  {
+		compatible = "intel,loapic";
+		reg = <0xfee00000 0x1000>;
+		num-irqs = <6>;
+		interrupt-controller;
+		#interrupt-cells = <3>;
+	};
+
+
```

Comment:
minor style thing: remove one of these empty lines

Reply:
Thanks! Resolved.